### PR TITLE
Feature/booking link

### PR DESCRIPTION
**Pull request by st3v3nhunt**
Adding the basics for bookOnlineLink logic and to get it displayed. Including the removal of the book-appointment route and view. Stuff left to do and should be ok to go in as additional PRs:

* Add `website` to the source data
* Handle no `contact.website` in the view - probably some wording

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-59.dev.c2s.nhschoices.net](http://profiles-pr-59.dev.c2s.nhschoices.net)

File: app/views/book-appointment.nunjucks [hunk 1]
```diff
@@ -1,115 +0,0 @@
...
-
-  <h2 id="closed">What to do when the surgery is closed</h2>
-  <p>coming soon</p>
-  <!-- <p>When the surgery is closed, you can still <a href="/gp-surgeries/book-appointment">book an appointment online</a>.</p> -->
-  <!-- <p>If you think you need to see someone more urgently, call 111 for advice on where to get help.</p> -->
-
-  <p><a href="{{ backLink }}" class="link-back">Back</a></p>
-
-</div>
-{% endblock %}
```
Comment (markysoft) [inline app/views/book-appointment.nunjucks:115]: The view has been removed, but there is still a reference to it in the renderer

**Review by markysoft — CHANGES_REQUESTED** [review]
Redundant function to remove

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-59.dev.c2s.nhschoices.net](http://profiles-pr-59.dev.c2s.nhschoices.net)

**Review by markysoft — APPROVED** [review]

**Pull request merged**